lib/chutney/TorNet.py

```diff
@@ -767,32 +766,30 @@ def _genRouterKey(self):
             sys.exit(1)
         self._env['fingerprint'] = fingerprint
        
-
     def _setEd25519Id(self):
         """Read the ed25519 identity key for this router, and set up the 'ed25519-id' entry in the Environ"""
         datadir = self._env['dir']
         key_directory = os.path.join(datadir, 'keys', "ed25519_master_id_public_key")
```

Comment:
This variable is the key file, not the key directory. (The key directory is `datadir/keys`.)

Can you rename it to `key_file`?